plugins/janus_audiobridge.c

```diff
@@ -2777,6 +2787,28 @@ void janus_audiobridge_incoming_rtp(janus_plugin_session *handle, int video, cha
 		pkt->silence = FALSE;
 		pkt->length = 0;
 
+		/* First check if probation period */
```

Comment:
What's the purpose of this probation, here? It looks like you're discarding some packets without ever decoding or mixing them. This seems to be happening whether FEC is in use or not, and I don't understand why.

Reply:
As you could see, I use expected sequence to implement FEC. So, you must be sure to start with a normal sequence (2 or 3 consecutive seq number) coming from one unique source. You must even go far from what I done here to be sure to get packet coming from only one distant source and also to detect a change in distant ssrc. It’s not important to drop 2 packets but more important is to start decoding correctly. It’s defense mechanism.

Reply:
The problem is that this is not done only for FEC, but even when FEC is disabled, which I don't think is the right way to go. Anyway, we can discuss this later on.